### PR TITLE
changes to arguments for vs2017+ compatibility

### PR DESCRIPTION
**Pull request by egoughnour**
Capitalizes first character for Framework45, etc., in cases where
necessary for compatibility. Also does not include Vsix-related argument in command line input where it is not recognized

### CODE REVIEW

File: src/main/resources/org/jenkinsci/plugins/vstest_runner/VsTestBuilder/help-useVs2017Plus_ja.html [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<div>
+    <p>
+        待ってください。
```
Comment (egoughnour) [inline src/main/resources/org/jenkinsci/plugins/vstest_runner/VsTestBuilder/help-useVs2017Plus_ja.html]: 私の日本語能力はかなり弱いですだからすみません。

**Comment by egoughnour** [comment]
This fixes [JENKINS-47081] and [JENKINS-47082].

File: src/main/java/org/jenkinsci/plugins/vstest_runner/VsTestBuilder.java [hunk 2]
```diff
@@ -118,7 +120,7 @@
      * @param failBuild
      */
     @DataBoundConstructor
-    public VsTestBuilder(String vsTestName, String testFiles, String settings, String tests, String testCaseFilter, boolean enablecodecoverage, boolean inIsolation, boolean useVsixExtensions, String platform, String otherPlatform, String framework, String otherFramework, String logger, String otherLogger, String cmdLineArgs, boolean failBuild) {
+    public VsTestBuilder(String vsTestName, String testFiles, String settings, String tests, String testCaseFilter, boolean enablecodecoverage, boolean inIsolation, boolean useVsixExtensions, boolean useVs2017Plus, String platform, String otherPlatform, String framework, String otherFramework, String logger, String otherLogger, String cmdLineArgs, boolean failBuild) {
```
Comment (daniel-beck) [inline src/main/java/org/jenkinsci/plugins/vstest_runner/VsTestBuilder.java]: Should probably just start using `@DataboundSetter` instead of this madness.

Comment (egoughnour) [reply src/main/java/org/jenkinsci/plugins/vstest_runner/VsTestBuilder.java]: Done.

**Comment by egoughnour** [comment]
@daniel-beck I don't see a merge drop down menu. Is there another way to merge PRs?

**Comment by egoughnour** [comment]
This change updates the arguments of VSTestAgent so that newer versions can also be used.
It also moves away from the many-parameter constructor.  
**help-useVs2017Plus_ja.html** may need some rewording.  I could not determine whether _argument_ should be used as a calque.  On the other hand I did not want to overuse 外来語 while losing accuracy by switching to _parameter_, for instance.

**Comment by egoughnour** [comment]
Trying to trigger issue resolution. [FIXED JENKINS-47081] and [FIXED JENKINS-47082]

**Pull request merged**